src/api/models.py

```diff
@@ -1,3 +1,35 @@
 from django.db import models
 
-# Create your models here.
+
+class Case(models.Model):
+
+	RELATIONSHIP_TYPE = [
+		(1, 'Current Spouse/Intimate Partner'),
+		(2, 'Former Spouse/Intimate Partner'),
+		(3, 'Current Dating Relationship'),
+		(4, 'Former Dating Relationship'),
+		(5, 'Other'),
+	]
+
+	RELATIONSHIP_LENGTH = [
+		(1, '<1 year'),
+		(2, '1-5 years'),
+		(3, '6-9 years'),
+		(4, '10-14 years'),
+		(5, '15-19 years'),
+		(6, '20-29 years'),
+		(7, '30+ years'),
+	]
```

Comment:
Can we maybe 0 index this? Other than that, this is beautiful ⭐⭐⭐

Reply:
agree with Julia's comment